<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessCreate</class>
 <widget class="QWidget" name="ProcessCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>进程创建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pid">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_neededtime">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>120</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>要求运行时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_priority">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pid">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_neededtime">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_priority">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_neededLength">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_neededLength">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>所需内存大小</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
